storage/sealer/storiface/storage.go

```diff
@@ -20,6 +20,17 @@ type SectorRef struct {
 
 var NoSectorRef = SectorRef{}
 
+// PieceNumber is a reference to a piece in the storage system; mapping between
+// pieces in the storage system and piece CIDs is maintained by the storage index
```

Comment:
why not use CIDs in the storage system?

Reply:
Mainly because it's a much much bigger change to the storage system (which today operates on actorid-number tuples).

There are other very slight advantages to number-based names, mostly that allow us to duplicate the same piece across storage easier if need be, but that is not the main reason.

Reply:
fwiw boost maintains a similar mapping of CommD->actor/sector/offset, here I'm just extending this model to work directly on pieces with minimal change to the storage system